hypervisor/arch/x86/e820.c

```diff
@@ -17,33 +17,34 @@
  * and hide HV memory from SOS_VM...
  */
 
-static uint32_t e820_entries_count;
-static struct e820_entry e820[E820_MAX_ENTRIES];
-static struct e820_mem_params e820_mem;
+static uint32_t hv_e820_entries_nr;
+static struct e820_entry hv_e820[E820_MAX_ENTRIES];
```

Comment:
Add  a comment like:  /* Describe the memory layout the hypervisor uses */

Do we need to change E820_MAX_ENTRIES too? is it for HV only, or for both HV & SOS?

Reply:
It's used for HV and all guests (SOS and UOS)